hubapi/deployment.py

```diff
@@ -44,7 +46,9 @@ def read_environment():
 @click.option('--deployment-stage',
               default='dev',
               help='Deployment stage for API Gateway (Default - dev)')
-def trigger(list_deployment_zip, push_deployment_zip, authorize_deployment_zip, key_id,
+@click.option('--docker-cred-deployment-zip',
```

Comment:
`click` reads env variables in the order decorator gets called & params are mentioned. Please correct this